cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,99 @@
-/// <reference types='cypress' />
+import { workerData } from '../support/userData';
+/// <reference types = "cypress" />
+
+function addWorker() {
+  const workerCount = 3;
+  Cypress._.times(workerCount, () => {
+    const newWorker = workerData();
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(newWorker.firstName);
+    cy.get('#lastName').type(newWorker.lastName);
+    cy.get('#userEmail').type(newWorker.email);
+    cy.get('#age').type('25');
+    cy.get('#salary').type(newWorker.salary);
+    cy.get('#department').type(newWorker.department);
+    cy.get('#submit').click();
+
+    cy.get('.rt-tbody').contains(`${newWorker.firstName}`).should('exist');
+
+    return newWorker;
+  });
+}
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('');
```

Comment:
The `cy.visit('');` command is missing a URL. You need to specify the URL of the page you want to visit before running the tests.